commands.go

```diff
@@ -70,7 +71,7 @@ func (w *commandWrapper) flush() {
 }
 
 func NewChrootCommandForContext(context DebosContext) Command {
```

Comment:
wonder if we should detect if the chrootmethod is supported here and error if not ?